index.bs

```diff
@@ -177,14 +183,15 @@ Files which contain one or more image sequences, and which conform with the prof
 
 Additionally, if the image sequences are made only of [=AV1 Samples=] marked as 'sync', then the brand <dfn value="" export="" for="AVIF Intra-only brand">avio</dfn> should be used.
 
+NOTE: As defined in [[!MIAF]], a file that is primarily an image sequence still has at least a primary image item. Hence, it can also declare brands for signaling the image item.
+
   <h2 id="profiles">Profiles</h2>
 
   <h3 id="profiles-overview">Overview</h3>
 
-  The profiles defined in this section are for enabling interoperability between [=AV1 Image File Format=] files and [=AV1 Image File Format=] readers/parsers.
-  A profile imposes a set of specific restrictions.
-  The presence of a brand indicating a profile can be interpreted as the permission for those [=AV1 Image File Format=] readers/parsers and [=AV1 Image File Format=] renderers that only implement the features required by the profile, to process the corresponding file.
-  An [=AV1 Image File Format=] file may conform to multiple profiles. Similarly, an [=AV1 Image File Format=] reader/parser or [=AV1 Image File Format=] renderer may be capable of processing one or more profiles.
+  The profiles defined in this section are for enabling interoperability between [=AV1 Image File Format=] files and [=AV1 Image File Format=] readers/parsers. A profile imposes a set of specific restrictions and is signaled by brands defined in this specification.
+
+  When auxiliary images or sequences are used, it is recommended to declare profiles via brands in the <code>FileTypeBox</code> that enable processing of at least the master image or sequence. That is, if the requirements of the profile of the auxiliary image or sequence are met by the processor needed for the associated master image, the brands for the auxiliary image or sequence should not be indicated in the <code>FileTypeBox</code>. If the auxiliary image or sequence uses a profile that requires more features than the associate master image or sequence, the brands in the <code>FileTypeBox</code> should reflect the requirements of the auxiliary image or sequence unless it is acceptable to process the file without the auxiliary image or sequence.
```

Comment:
> This paragraph feels a bit too verbose.

I kind of agree.

> The first sentence says to declare profiles in the ftyp box.

Not exactly. It says use the FileTypeBox (as opposed to using only the TrackTypeBox) and use it to signal the requirements of the main content (not the alpha one). 

> The second sentence tries to clarify the first sentence. The third sentence has the key information.

The 2nd and 3rd sentences try to clarify the first sentence by detailing the 2 sub-cases: 
- the alpha content profile is stricter than the main content profile: declare the main content profile in the FileTypeBox and the alpha one in the TrackTypeBox.
- the main content profile is stricter than the alpha content profile: declare only the main content profile in the FileTypeBox


> What would happen if we simplify this paragraph to just have the third sentence? Is any crucial information lost?

I'd rather keep only the first sentence, but I'm open to any shorter rewrite.

Reply:
I am bit more confused now. By "stricter" do you mean "requires fewer features" than another profile? For example, Basic profile would be "stricter" than the Advanced profile, because Basic profile allows for fewer features than the Advanced profile.
What I think we need to say is that what goes in the 'ftyp' box is the profile that has all the required features to display both images.
Example 1: If the main image uses Basic profile and the alpha image uses Advanced profile, then Advanced profile should be listed in the 'ftyp' box, because you won't be able to properly display the full image (main image + alpha) unless you have support for Advanced profile. (Support for Advanced profile implies support for Basic profile.)
Example 2: If the main image uses Advanced profile, and the alpha image uses Basic profile, then Advanced profile should be listed in the 'ftyp' box, for the same reason as in example 1.

Reply:
> By "stricter" do you mean "requires fewer features" than another profile? For example, Basic profile would be "stricter" than the Advanced profile, because Basic profile allows for fewer features than the Advanced profile.

Yes



> What I think we need to say is that what goes in the 'ftyp' box is the profile that has all the required features to display both images.

I almost agree. Sometimes you don't care if the auxiliary image cannot be displayed, in which case it would be ok to advertise only the main image profile. That's the intent of the 3rd sentence:
> unless it is acceptable to process the file without the auxiliary image or sequence.

Reply:
> Sometimes you don't care if the auxiliary image cannot be displayed, in which case it would be ok to advertise only the main image profile. 

OK, that's fine. So, we are in agreement then. I have a proposal for how the paragraph can be simplified (perhaps) using multiple shorter sentences. There are a couple of issues I want to point out first:
- The term "master image" isn't right in this context. I think what we really want to say is "primary image, or one of its alternates", because the file could contain a bunch of different master images and the 'ftyp' box is really about the primary image only.
- Let's use the term "decode" rather than the more vague "process".
- In case of an Image Sequence file, the brand needs to apply both to the primary image and one of the non-auxiliary tracks. (Wouldn't be right to have a file with 'avis' brand that says the profile is Basic if only the primary image requires Basic profile but the track requires Advanced profile.)
- Note that for a given master image there can be more than one auxiliary image -- for example, both Alpha and Depth at the same time.

This gets a bit tricky to explain without having to declare a new term, such as "selected item" or "selected track". In any case, here is my proposal:

"The FileTypeBox should declare at least one profile that enables decoding of the primary image item, or one of its alternates.
The profile shall allow decoding of any associated auxiliary images, unless it is acceptable to decode the image item without its auxiliary images.

If an image sequence brand is declared in the FileTypeBox, then the profile shall also enable decoding of at least one image sequence track.
The profile shall allow decoding of any associated auxiliary image sequence tracks, unless it is acceptable to decode the image sequence without its auxiliary image sequence tracks."

Reply:
I've used a slightly modified version of your text. Changing 'shall' to 'should' when followed by 'except'. Hopefully we can merge now. Let me know.